boards/arm/mimxrt1170_evk/doc/index.rst

```diff
@@ -0,0 +1,280 @@
+core i.MX RT1170 runs on the Cortex-M7 core at 1 GHz and Arm Cortex-M4 at
+400 MHz, while providing best-in-class security. The i.MX RT1170 MCU offers
+support over a wide temperature range and is qualified for consumer,
+industrial and automotive markets.
+
+.. image:: ./mimxrt1170_evk.jpg
+   :width: 600px
+   :align: center
+   :alt: MIMXRT1170-EVK
+
+Hardware
+********
+
+- MIMXRT1176DVMAA MCU
+
+  - 1GHz Cortex-M7 & 400Mhz Cortex-M4
+  - 2MB SRAM with 512KB of TCM for Cortex-M7 and 256KB of TCM for Cortex-M4
+
+- Memory
+
+  - 512 Mbit SDRAM
+  - 128 Mbit QSPI Flash
+  - 512 Mbit Octal Flash
+  - 2 Gbit raw NAND flash
+  - 64 Mbit LPSPI flash
+  - TF socket for SD card
+
+- Display
+
+  - MIPI LCD connector
+
+- Ethernet
+
+  - 10/100 Mbit/s Ethernet PHY
+  - 10/100/1000 Mbit/s Ethernet PHY
+
+- USB
+
+  - USB 2.0 OTG connector
+  - USB 2.0 host connector
+
+- Audio
+
+  - 3.5 mm audio stereo headphone jack
+  - Board-mounted microphone
+  - Left and right speaker out connectors
+
+- Power
+
+  - 5 V DC jack
+
+- Debug
+
+  - JTAG 20-pin connector
+  - OpenSDA with DAPLink
+
+- Sensor
+
+  - FXOS8700CQ 6-axis e-compass
+  - MIPI camera sensor connector
+
+- Expansion port
+
+  - Arduino interface
+
+- CAN bus connector
+
+For more information about the MIMXRT1170 SoC and MIMXRT1170-EVK board, see
+these references:
+
+- `i.MX RT1170 Website`_
+- `i.MX RT1170 Datasheet`_
+- `i.MX RT1170 Reference Manual`_
+- `MIMXRT1170-EVK Website`_
+- `MIMXRT1170-EVK Board Hardware User's Guide`_
+
+Supported Features
+==================
+
+The mimxrt1170_evk board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| DISPLAY   | on-chip    | display                             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| SDHC      | on-chip    | disk access                         |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| ENET      | on-chip    | ethernet                            |
++-----------+------------+-------------------------------------+
+| USB       | on-chip    | USB device controller               |
++-----------+------------+-------------------------------------+
```

Comment:
Can you confirm USB, ENET, SDHC, DISPLAY are enabled. I don't see these in the dts file

Reply:
I have checked index.rst file again and deleted peripherals disabled.